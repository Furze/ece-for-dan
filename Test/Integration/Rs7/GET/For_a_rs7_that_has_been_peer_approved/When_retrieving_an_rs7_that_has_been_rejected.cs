using Bard;
using MoE.ECE.Domain.Event;
using MoE.ECE.Domain.Model.ValueObject;
using MoE.ECE.Domain.Read.Model.Rs7;
using MoE.ECE.Integration.Tests.Chapter;
using MoE.ECE.Integration.Tests.Infrastructure;
using Shouldly;
using Xunit;
using Xunit.Abstractions;

namespace MoE.ECE.Integration.Tests.Rs7.GET.For_a_rs7_that_has_been_peer_approved
{
    // ReSharper disable once InconsistentNaming
    public class When_retrieving_an_rs7_that_has_been_rejected : SpeedyIntegrationTestBase
    {
        public When_retrieving_an_rs7_that_has_been_rejected(RunOnceBeforeAllTests testSetUp,
            ITestOutputHelper output, TestState<ECEStoryBook, ECEStoryData> testState) : base(testSetUp, output,
            testState)
        {
        }

        private Rs7Model Rs7Model
        {
            get => TestData.Rs7Model;
            set => TestData.Rs7Model = value;
        }

        protected override void Arrange()
        {
            Given
                .A_rs7_has_been_created()
                .The_rs7_has_been_submitted_for_peer_approval()
                .The_rs7_has_been_peer_review_rejected()
                .GetResult(storyData => Rs7Model = storyData.Rs7Model);
        }

        protected override void Act()
        {
            When.Get($"{Constants.Url}/{Rs7Model.Id}");
        }

        [Fact]
        public void Domain_event_raised()
        {
            A_domain_event_should_be_fired<Rs7PeerRejected>();
        }

        [Fact]
        public void Roll_status_has_been_updated()
        {
            Then.Response.Content<Rs7Model>().RollStatus.ShouldBe(RollStatus.ExternalReturnedForEdit);
        }
    }
}