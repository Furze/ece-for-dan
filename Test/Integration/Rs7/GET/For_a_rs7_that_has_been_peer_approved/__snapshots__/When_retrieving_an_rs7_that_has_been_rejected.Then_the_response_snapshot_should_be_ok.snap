{
  "Id": 17008,
  "BusinessEntityId": "c11f6047-12d3-4796-8f2f-7c07cb7dffba",
  "OrganisationId": 114895,
  "FundingPeriod": "July",
  "FundingYear": 2021,
  "FundingPeriodYear": 2020,
  "IsZeroReturn": false,
  "RollStatus": "ExternalReturnedForEdit",
  "ReceivedDate": "2020-11-13T10:01:10.2139471+13:00",
  "RevisionNumber": 1,
  "RevisionDate": "2020-11-13T10:01:10.3451736+13:00",
  "AdvanceMonths": [
    {
      "MonthNumber": 7,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "MonthNumber": 8,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "MonthNumber": 9,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    },
    {
      "MonthNumber": 10,
      "Year": 2020,
      "AllDay": 2,
      "Sessional": null,
      "ParentLed": null
    }
  ],
  "EntitlementMonths": [
    {
      "MonthNumber": 2,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "MonthNumber": 3,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 31,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "MonthNumber": 4,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    },
    {
      "MonthNumber": 5,
      "Year": 2020,
      "Days": [
        {
          "DayNumber": 1,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 2,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 3,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 4,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 5,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 6,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 7,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 8,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 9,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 10,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 11,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 12,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 13,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 14,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 15,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 16,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 17,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 18,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 19,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 20,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 21,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 22,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 23,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 24,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 25,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 26,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 27,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 28,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 29,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 30,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        },
        {
          "DayNumber": 31,
          "Under2": null,
          "TwoAndOver": null,
          "Hours20": null,
          "Plus10": null,
          "Certificated": null,
          "NonCertificated": null
        }
      ]
    }
  ],
  "IsAttested": true,
  "Declaration": null,
  "RequestId": "17008"
}
